test/test_torch.py

```diff
@@ -10100,6 +10100,52 @@ def gen_error_message(input_size, p, keepdim, dim=None):
             with self.assertRaisesRegex(RuntimeError, r'"norm_cuda" not implemented for \'ComplexFloat\''):
                 (1j * torch.randn(25)).norm()
 
+    # Ensure torch.norm with p='fro' and p=2 give the same results for mutually supported input combinations
+    @dtypes(torch.float, torch.double)
```

Comment:
Would you cut + paste this (and, while we're at it, the other norm tests) into test_linalg.py?